common-jvm/src/main/java/org/ergoplatform/transactions/ColdWalletUtils.kt

```diff
@@ -66,13 +54,10 @@ fun parseColdSigningRequest(qrData: String): PromptSigningResult {
         val serializedTx = Base64Coder.decode(jsonTree.get(JSON_FIELD_REDUCED_TX).asString)
 
         // sender is optional
-        val sender =
-            if (jsonTree.has(JSON_FIELD_SENDER)) jsonTree.get(JSON_FIELD_SENDER).asString else null
+        val sender = jsonTree.get(JSON_FIELD_SENDER)?.asString
 
-        val inputs = if (jsonTree.has(JSON_FIELD_INPUTS)) {
-            jsonTree.get(JSON_FIELD_INPUTS).asJsonArray.toList()
-                .map { Base64Coder.decode(it.asString) }
-        } else null
+        val inputs = jsonTree.get(JSON_FIELD_INPUTS)?.asJsonArray?.toList()
+                ?.map { Base64Coder.decode(it.asString) }
```

Comment:
What would you suggest to encapsulate here?
Unfortunately Gson is not very Kotlin-friendly anyway. Using kotlinx-serialization would make better code here. But we already have Gson pulled in, and pulling in a second JSON parser is something I want to avoid.

Reply:
I keep something like this in mind
`jsonTree.decodeBase64StringList(JSON_FIELD_INPUTS)`

Reply:
Changed it, although I am not sure as this is a very specific method